expression.go

```diff
@@ -81,6 +81,18 @@ func (expr ValueExpr) RootRefs() []FieldExpr {
 	return nil
 }
 
+type RegexExpr struct {
+	Regex interface{}
+}
+
+func (expr RegexExpr) String() string {
+	return fmt.Sprintf("%v", expr.Regex)
```

Comment:
Maybe we should differentiate this from a normal string, perhaps using "/%v/"?

Reply:
Will make the change